app/templates/drupal/octane/8.x/composer.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
I'd like to see https://github.com/phase2/generator-gadget/pull/66 settle first, then we can either copy that config over to Octane or look at having a central template resource for composer.json in D8.